01 Operator Overloading/Operator Overloading.Host/Program.cs

```diff
@@ -13,28 +14,26 @@ static void Main(string[] args)
         {
             try
             {
-                Console.WriteLine("Enter 1st Amount : ");
-
+                Console.WriteLine("Format : 100 USD");
+                Console.WriteLine("\nEnter Amount : "); // Enter Amount in Format '100 USD'
                 var m1 = new Money(Console.ReadLine());
 
+                Console.WriteLine("\nEnter target Currency : "); // Enter target currency
+                var m2 = new Money(0, Console.ReadLine());
```

Comment:
Why did you create money object here? `Convert` method will take `string` and will return a `Money` object.